src/convertToObject.js

```diff
@@ -6,7 +6,34 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.replaceAll('\n', ' ').split(';');
+  const filtredStylesList = stylesList.filter((val) => val.trim().length > 0);
+
+  const segregatedList = [];
+  const stylesObject = {};
+
+  function segregate(el) {
+    const splited = el.split(':');
```

Comment:
The current implementation does not handle multiple colons within a style string's property values. Consider adding a check to ensure the split results in exactly two parts to make the code more robust.